nixos/modules/services/web-apps/nextcloud-notify_push.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.nextcloud.notify_push;
+in
+{
+  options.services.nextcloud.notify_push = {
+    enable = lib.mkEnableOption "Notify push";
```

Comment:
This should mention that this has a hard requirement on nextcloud.

Reply:
It's under the nextcloud namespace.

Reply:
Oh, overread that